### ISSUE TITLE
Test and validate B3DM Batch Table and Feature Table support

### ISSUE THREAD

**Issue by gkjohnson** [NASA-AMMOS/3DTilesRendererJS#63]
See example files [here](https://github.com/CesiumGS/3d-tiles-samples/tree/master/tilesets/TilesetWithRequestVolume/city).

**Comment by gkjohnson** [comment]
Looks like we should also set the website to scale properly and work with mobile:

```html
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
```

**Comment by gkjohnson** [comment]
### Feature Table

Describes general pieces of data or "features" within a model, such as a building:

https://github.com/CesiumGS/3d-tiles/tree/master/specification/TileFormats/FeatureTable

Every model / feature / vertex may get a batch id to associate it with information in the batch table.

### Batch Tables

Per-batch id information -- batch ids are associated with different features.

https://github.com/CesiumGS/3d-tiles/tree/master/specification/TileFormats/BatchTable

### B3DM

**Feature Table**

Describes the number of batch ids in the batch table.

### I3DM

**Feature Table**

Describes the number of instances.

### PNTS

**Feature Table**

Describes the points.

**Comment by gkjohnson** [comment]
Implemented and verified in #81.

Closing in favor of #65, #68
